src/PDS4_CHAN1_IngestLDD.xml

```diff
@@ -77,8 +80,8 @@
             <enumeration_flag>true</enumeration_flag>
             <value_data_type>ASCII_Short_String_Collapsed</value_data_type>
             <DD_Permissible_Value>
-                <value>Nominal Mission</value>
```

Comment:
Looks like this will need to keep options "Nominal" and "Primary". This mission dictionary also need to work with Mini-RF: https://pds-geosciences.wustl.edu/lunar/ch1-orb-l-mrffr-1-pdr-v1/ch1mrf_0xxx/data/sar/00700_00799/level1/fsb_00720_1cd_xhu_84n209_v1.xml

Reply:
ok, I will add "Nominal" back in. Looks like Mini-RF used "Nominal" for all their products and M3 "Primary", for whatever reason